Source/AwsCommonRuntimeKit/auth/credentials/CredentialsProvider.swift

```diff
@@ -76,7 +138,7 @@ extension CredentialsProvider {
 }
 
 extension CredentialsProvider.Source {
-
+    
```

Comment:
There are a large number of weird white space additions onto empty lines. Is this an auto formatting issue?